internal/postgres/config.go

```diff
@@ -233,28 +255,15 @@ while read -r -t 5 -u "${fd}" ||:; do
     exec {fd}>&- && exec {fd}<> <(:||:)
     stat --format='Loaded certificates dated %%y' "${directory}"
   fi
-
-  # Manage autogrow annotation.
-  # Return size in Mebibytes.
-  size=$(df --human-readable --block-size=M /pgdata | awk 'FNR == 2 {print $2}')
-  use=$(df --human-readable /pgdata | awk 'FNR == 2 {print $5}')
-  sizeInt="${size//M/}"
-  # Use the sed punctuation class, because the shell will not accept the percent sign in an expansion.
-  useInt=$(echo $use | sed 's/[[:punct:]]//g')
-  triggerExpansion="$((useInt > 75))"
-  if [ $triggerExpansion -eq 1 ]; then
-    newSize="$(((sizeInt / 2)+sizeInt))"
-    newSizeMi="${newSize}Mi"
-    d='[{"op": "add", "path": "/metadata/annotations/suggested-pgdata-pvc-size", "value": "'"$newSizeMi"'"}]'
-    curl --cacert ${CACERT} --header "Authorization: Bearer ${TOKEN}" -XPATCH "${APISERVER}/api/v1/namespaces/${NAMESPACE}/pods/${HOSTNAME}?fieldManager=kubectl-annotate" -H "Content-Type: application/json-patch+json" --data "$d"
-  fi
```

Comment:
I think it would be nice to add a unit test with the FG enabled so that we can assert that is part exists in the script. WDYT?